torch/multiprocessing/__init__.py

```diff
@@ -34,10 +34,9 @@
     from .pool import Pool
 
 
-if sys.version_info >= (3, 4):
-    """Add helper function to spawn N processes and wait for completion of any of
-    them. This depends `mp.get_context` which was added in Python 3.4."""
-    from .spawn import spawn, SpawnContext
+"""Add helper function to spawn N processes and wait for completion of any of
+them. This depends `mp.get_context` which was added in Python 3.4."""
+from .spawn import spawn, SpawnContext
```

Comment:
Comment above can now go since it's no longer conditional.

Reply:
I decided to keep it since it's for informative